### ISSUE TITLE
Lab 2 

### ISSUE THREAD

**Issue by sunaynagoel** [DS4PS/cpp-523-fall-2019#4]
I am stuck in lab 2 . I may be overthinking, I really don't know how to make sense of the  table provided in lab 02. I went through all the study materials and watch some videos tutorials online as well. 

Please help.

**Comment by lecy** [comment]
For a single regression the formula maps to the table like this:

![image](https://user-images.githubusercontent.com/1209099/63985957-b6094500-ca86-11e9-8ec4-fed142f836d4.png)

**Comment by sunaynagoel** [comment]
Thank you this helps a lot. How would I interpret x dash or sample mean for x ?

**Comment by lecy** [comment]
When you are comparing models we use the compact table with one column for each model:

![image](https://user-images.githubusercontent.com/1209099/63986099-58292d00-ca87-11e9-9c33-660ed91df7a6.png)

**Comment by sunaynagoel** [comment]
So sorry for being a pain. I do understand the concept on Week 2 and I get Lab 02 concept as well. It just the Math I am getting stuck. For example question 3, for finding CI I think I am missing one information about x dash. i am not sure if I am right?

**Comment by lecy** [comment]
Ah, these are confidence intervals for the slopes. 

**Q.** Calculate the 95% confidence interval around the class size coefficient in Model 1. 

```
CI.upper = b1  +  t * SE(b1)
CI.lower = b1  -  t * SE(b1)
```

In the experimental context we would compare the mean of the treatment GROUP and mean of the control GROUP. Our hypothesis test would require us to calculate x-bar (the sample mean) in that case. 

In the regression context all of the confidence intervals will be around the SLOPES, not the sample means. You will use the formulas above.

**Comment by sunaynagoel** [comment]
This makes so much more sense. I was overthinking in terms of math. Thanks a lot

**Comment by lecy** [comment]
No problem! I see why your mind would go down that path. Thanks for checking.

**Comment by brittanyharb** [comment]
My question on this is what/where can I find the t variable? I've done lots of research and am still struggling to establish this

**Comment by Jigarci3** [comment]
Hello,

I am also feeling stuck trying to make sense of the model. 

For question 1 - I understand that the formula for the error of the slope is:

Standard error of the slope= residual/Sample size ⋅ Variance x

I need help understanding how to read the model please.

**Comment by lecy** [comment]
The instructions at the top say:

> For the following cases, use the value t=1.96 for the 95% confidence interval calculations.

This is a topic usually covered in intro stats that I don't feel is very useful to spend time on because the computer will automatically calculate this value in most models behind the scenes, so you never actually look it up in practice. And in policy we typically do not do studies with samples small enough to use another t-value. We will use the 1.96 value throughout this course unless otherwise stated.

**Comment by brittanyharb** [comment]
Oh shoot. Thank you for clarifying. My apologies, I guess I didn't read the instructions thoroughly

**Comment by lecy** [comment]
Joanna - if you scroll down in the lab I have included a diagrammed table. Does that help? 

![image](https://user-images.githubusercontent.com/1209099/64137476-490aed80-cdad-11e9-8b37-744fdcd64c5f.png)

Question 1 is asking you to read the table only, not calculate the value yourself. 

Different disciplines report different stats in parentheses, so I wanted to make sure it was clear that the standard errors are reported in the table, not p-values or t-stats.

**Comment by Jigarci3** [comment]
I see... thank you for clarifying.

**Comment by brittanyharb** [comment]
So, just checking, an upper and lower confidence interval with negative values after applying the numbers into the equation is normal sometimes? If the relationship of x and y co-variants is negative in nature as well?

**Comment by lecy** [comment]
Yeah it's a little counter-intuitive when the program has a negative slope, but in this case smaller classes achieve better outcomes, so your math is correct. 

![image](https://user-images.githubusercontent.com/1209099/64138972-292af800-cdb4-11e9-935b-2ca353e324e8.png)

**Comment by lecy** [comment]
@sunaynagoel Please don't close issues because that hides the responses from other students. You can just reply with a "that worked" to confirm that the issue was resolved, but the information might be useful to someone else that is having the same problem. Thanks!

**Comment by sunaynagoel** [comment]
Sure.. I was just thinking it is my responsibility to close an issue if I was the one to open it. But yes this makes sense. I always go in check for responses so many times myself.
